HLTrigger/Configuration/python/Tools/confdbOfflineConverter.py

```diff
@@ -288,7 +308,20 @@ def main():
     if '--dbproxyport' in args:
         proxy_port = args.pop(args.index('--dbproxyport')+1)
         args.remove('--dbproxyport')
-        
+
+    tunnel = False
+    tunnel_port = "10121"
```

Comment:
Note to self (not for this PR): we should find a way to set the default values only once (e.g. in `options.py`), without having to hard-code more than once.